### PR TITLE
Add service name tracking and include in export

### PR DESCRIPTION
**Pull request by ChristineDuong**
Keeps track of what matching services returns a trial. Returns that in the export. Also filters out pilot trial.

### CODE REVIEW

File: src/components/Results/utils.ts [hunk 1]
```diff
@@ -200,15 +200,15 @@ const getClosestFacilities = (locations: Location[], zipcode: string, numOfFacil
...
+export const getStudyDetailProps = (entry: BundleEntry, zipcode: string, serviceName: string): StudyDetailProps => {
   const { resource, search } = entry;
   // Grab the locations so we only have to do this once
   const locations = getLocations(resource);
 
   return {
     conditions: getConditions(resource),
     trialId: getTrialId(resource),
-    source: getSource(),
+    source: serviceName,
```
Comment (dmpotter44) [inline src/components/Results/utils.ts:211]: It looks like this removes the use of `getSource()` (added on line 47, which for whatever reason, I can't add a comment to), which should probably be removed.

File: src/pages/api/clinical-trial-search.ts [hunk 2]
```diff
@@ -125,8 +125,7 @@ async function callWrappers(matchingServices: string[], query: Bundle, patientZi
   const errors = wrapperResults.filter(result => result.status == 500);
 
   // Combine the responses that were successful
-  const combined: StudyDetailProps[] = [];
-  const uniqueTrialIds = new Set<string>();
+  const uniqueTrials: { [key: string]: StudyDetailProps } = {};
```
Comment (dmpotter44) [inline src/pages/api/clinical-trial-search.ts:128]: I'm tempted to say this should really be a `Map<string, StudyDetailProps>`, but this almost certainly doesn't really matter. (The only reason to do that would be if we were likely to get keys that interfered with built-in JavaScript names, and even if we do, it shouldn't really matter.)

**Review by dmpotter44 — COMMENTED** [review]
Really only one comment, `getSource()` should probably be removed if it's no longer used. It looks like its been effectively implemented so it should be safe to remove it and the associated TODO.

**Review by dmpotter44 — APPROVED** [review]

File: src/pages/api/clinical-trial-search.ts (hunk 2 above)
Comment (ChristineDuong) [reply src/pages/api/clinical-trial-search.ts:128]: I don't mind switching it to a `Map`, but the keys should this _should_ only be NCT ids so there shouldn't be any conflict.

**Pull request merged**